src/random_test.cpp

```diff
@@ -1608,28 +1609,92 @@ void Table::DeleteRandomRow(Thd1 *thd) {
     }
   }
 
-  std::string sql = "DELETE FROM " + name_ + " WHERE " +
-                    columns_->at(where)->name_ + "=" +
+  std::string sql1 = "DELETE FROM " + name_ + " WHERE " +
+                    columns_->at(where)->name_ + " = " +
                     columns_->at(where)->rand_value();
+  std::string sql2 = "DELETE FROM " + name_ + " WHERE " +
+                    columns_->at(where)->name_ + " >= " +
+                    columns_->at(where)->rand_value() + " AND " +
+                    columns_->at(where)->name_ + " <= " +
+                    columns_->at(where)->rand_value();
+  std::string sql3 = "DELETE FROM " + name_ + " WHERE " +
```

Comment:
it could either end up deleting no row or all rows.
assign it very less probability. as compare to = and like.

Reply:
Done